ledger-core/v2/conveyor/pulse_manager.go

```diff
@@ -203,9 +203,9 @@ func (p *PulseDataManager) PreparePulseDataRequest(ctx smachine.ExecutionContext
 ) smachine.AsyncCallRequester {
 	switch {
 	case resultFn == nil:
-		panic("illegal value")
+		panic("illegal value: empty resultFn")
 	case p.pulseDataAdapterFn == nil:
-		panic("illegal state")
+		panic("illegal state: PulseDataServicePrepareFunc")
```

Comment:
Please don't do this - location of panic is already sufficiently descriptive. It's like noisy comments

Reply:
removed